academic-services/knowledge-exploration-service/concepts-query-expressions.md

```diff
@@ -1,9 +1,91 @@
 ---
-title: Structured Query Expressions
-description: 
+title: Structured query expressions
+description: Learn how to use structured query expressions
 ms.topic: reference
-ms.date: 2020-02-07
+ms.date: 2020-02-10
 ---
 
-# Structured Query Expressions
+# Structured query expressions
 
+A structured query expression specifies a set of operations to evaluate against the index for the purpose of retrieving entities.  It consists of both attribute query expressions and higher-level functions.  Use the [Evaluate method](reference-get-evaluate.md) to retrieve matching entities for a query expression.  
```

Comment:
Doesn't the calchistogram method use query expressions as well?  Why single out the Evaluate expression?